### DISCUSSION TITLE
Scavenge feature suggestion

### DISCUSSION THREAD

**Discussion by TheWater1105** [Zrips/CMI#6957] [Feature Requests]
Scavenge should be able to only extracts enchantments,because players can get diamonds by trading diamonds tools and equipment with villagers. I want players to only extract enchantments instead of break items. I really need this feature, thank you.

**Comment by Zrips** [comment]
Set item brake to 100%, ingredient return to 0% and then play around with enchantment extraction rates depending on your needs. 
Or are you just want to avoid braking items but still extra enchantments? If so, you could set item brake to 0%, ingredient return percentage wont really have any effect if item doesnt brake, and then simply set enchant extraction rates to your liking.
Unless I missed behavior of scavenge you actually want.

**Reply by TheWater1105** [reply]
Thank you!
